rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -463,134 +496,133 @@ namespace rocal{
+            .value("DECODER_VIDEO_FFMPEG_SW", ROCAL_DECODER_VIDEO_FFMPEG_SW)
+            .value("DECODER_VIDEO_FFMPEG_HW", ROCAL_DECODER_VIDEO_FFMPEG_HW)
             .export_values();
         // rocal_api_info.h
-        m.def("getRemainingImages",&rocalGetRemainingImages);
-        m.def("getImageName",&wrapper_image_name);
+        m.def("getRemainingImages", &rocalGetRemainingImages);
+        m.def("getImageName", &wrapper_image_name);
         m.def("getImageId", [](RocalContext context, py::array_t<int> array) {
             auto buf = array.request();
-            int* ptr = (int*) buf.ptr;
+            int* ptr = static_cast<int *>(buf.ptr);
             return rocalGetImageId(context, ptr);
         });
-        m.def("getImageNameLen",&wrapper_image_name_length);
-        m.def("getStatus",&rocalGetStatus);
-        m.def("setOutputImages",&rocalSetOutputs);
-        m.def("labelReader",&rocalCreateLabelReader);
-        m.def("TFReader",&rocalCreateTFReader);
-        m.def("TFReaderDetection",&rocalCreateTFReaderDetection);
-        m.def("CaffeReader",&rocalCreateCaffeLMDBLabelReader);
-        m.def("Caffe2Reader",&rocalCreateCaffe2LMDBLabelReader);
-        m.def("CaffeReaderDetection",&rocalCreateCaffeLMDBReaderDetection);
-        m.def("Caffe2ReaderDetection",&rocalCreateCaffe2LMDBReaderDetection);
+        m.def("getImageNameLen", &wrapper_image_name_length);
+        m.def("getStatus", &rocalGetStatus);
+        m.def("setOutputs", &rocalSetOutputs);
+        m.def("tfReader", &rocalCreateTFReader, py::return_value_policy::reference);
+        m.def("tfReaderDetection", &rocalCreateTFReaderDetection, py::return_value_policy::reference);
+        m.def("caffeReader", &rocalCreateCaffeLMDBLabelReader, py::return_value_policy::reference);
+        m.def("caffe2Reader", &rocalCreateCaffe2LMDBLabelReader, py::return_value_policy::reference);
+        m.def("caffeReaderDetection", &rocalCreateCaffeLMDBReaderDetection, py::return_value_policy::reference);
+        m.def("caffe2ReaderDetection", &rocalCreateCaffe2LMDBReaderDetection, py::return_value_policy::reference);
         m.def("isEmpty", &rocalIsEmpty);
         m.def("getStatus", rocalGetStatus);
         m.def("rocalGetErrorMessage", &rocalGetErrorMessage);
-        m.def("rocalGetTimingInfo", &rocalGetTimingInfo);
         m.def("getTimingInfo", &rocalGetTimingInfo);
-        m.def("setOutputImages", &rocalSetOutputs);
         m.def("labelReader", &rocalCreateLabelReader, py::return_value_policy::reference);
-        m.def("COCOReader", &rocalCreateCOCOReader, py::return_value_policy::reference);
+        m.def("cocoReader", &rocalCreateCOCOReader, py::return_value_policy::reference);
         // rocal_api_meta_data.h
-        m.def("RandomBBoxCrop", &rocalRandomBBoxCrop);
-        m.def("BoxEncoder",&rocalBoxEncoder);
+        m.def("randomBBoxCrop", &rocalRandomBBoxCrop);
+        m.def("boxEncoder", &rocalBoxEncoder);
```

Comment:
Remove commented line below

Reply:
Or add a comment that this would be uncommented when the IOU matcher example would be introduced